src/components/Banner/Banner.module.css

```
@@ -10,27 +10,21 @@
  * styles for the horizontal orientation below
  */
 .banner {
-  display: grid;
-  gap: var(--eds-size-2);
-  position: relative;
 
+  @apply grid gap-4 relative;
   background-color: var(--eds-theme-color-neutral-subtle-background);
   /**
    * Has a bottom margin to accommodate drop shadow
    */
-  box-shadow: var(--eds-box-shadow-md);
-  margin-bottom: var(--eds-size-2);
 
-  border-style: solid;
-  border-radius: var(--eds-border-radius-md);
-  border-width: var(--eds-theme-border-width);
-  border-color: var(--messaging-border-color);
+  
+  box-shadow: var(--eds-box-shadow-md); /* 0px 0px 2px rgba(0, 0, 0, 0.2), 0px 2px 8px rgba(0, 0, 0, 0.08) */
+  margin-bottom: var(--eds-size-2); /* 1rem */
+  @apply border-solid rounded border-[1px];
```

Does `border` with no number work for the 1px width?
```suggestion
  @apply border-solid rounded border;
```

---

@dierat As per tailwind css, for border width of 1px, we can use just border